src/authkit-callback-route.ts

```diff
@@ -48,7 +61,6 @@ export function authLoader(options: HandleAuthOptions = {}) {
           refreshToken,
           user,
           impersonator,
-          oauthTokens,
```

Comment:
Github won't let me add a comment there, but on line 57 we still mention `oauthTokens`.